distmaker/dists/standalone.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+
+P=`dirname $0`
+CFFILE=$P/../distmaker.conf
+if [ ! -f $CFFILE ] ; then
+	echo "NO DISTMAKER.CONF FILE!"
+	exit 1
+else
+	. $CFFILE
+fi
+. "$P/common.sh"
+
+SRC=$DM_SOURCEDIR
+TRG=$DM_TMPDIR/civicrm
+
+# copy all the stuff
+dm_reset_dirs "$TRG" "$TRG/civicrm/core"
+dm_install_core "$SRC" "$TRG/civicrm/core"
+dm_install_coreext "$SRC" "$TRG/civicrm/core" $(dm_core_exts)
+dm_install_packages "$SRC/packages" "$TRG/civicrm/core/packages"
+dm_install_vendor "$SRC/vendor" "$TRG/civicrm/core/vendor"
+dm_install_bower "$SRC/bower_components" "$TRG/civicrm/core/bower_components"
+dm_install_cvext com.iatspayments.civicrm "$TRG/civicrm/core/ext/iatspayments"
```

Comment:
Do we need to include the standalone extension?

Reply:
@seamuslee001 it's part of core: https://github.com/civicrm/civicrm-core/tree/master/ext/standaloneusers

Reply:
In this file, it has:

```bash
dm_install_coreext "$SRC" "$TRG/civicrm/core" $(dm_core_exts)
```

which reads `distmaker/core-ext.txt` -- and yes, `standaloneusers` is listed there too. ✅ 

(Fun aside: I think `standaloneusers` was added to `core-ext.txt` to ensure that `set-version.php` maintains it. However, this does imply that *all* tarballs include it. That's a bit extraneous. But as long the ext is hidden, it doesn't feel like a real problem.)

Reply:
@totten so should this be merged?

Reply:
@colemanw Not yet. (a) I don't think anyon has `r-run` the output. (b) it probably needs [some change to serve static assets](https://github.com/civicrm/civicrm-core/pull/27104#issuecomment-1687326121) -- e.g. use the `web/core/` folder or revise `htaccess`.

Reply:
(But this subthread can be resolved. I ran distmaker and confirmed that `ext/standaloneusers`  is included.)